source/fundamentals/aggregation/aggregation-expression-operations.txt

```diff
@@ -421,15 +453,19 @@ the Query API:
        $project: {
          extremeTemp: { $or: [ { $lt: ["$temperature", 10] },
                                { $gt: ["$temperature", 95] },
-           ] } }
+         ] }
+       }
      }
    ]
 
-Conversion Operations
+Comparison Operations
 ~~~~~~~~~~~~~~~~~~~~~
 
-You can perform a conversion operation to convert between ``MqlValue``
-types using the methods described in this section.
+You can perform a comparison operation on a value of type ``MqlValue``
+using the methods described in this section. For more information about how
+the driver compares values of different types, see
+:manual:`Aggregation Pipeline Operators </reference/bson-type-comparison-order/#std-label-bson-types-comparison-order>`
+in the Server manual.
```

Comment:
There is not a one-to-one mapping between categories. I can check to see if there are other sections that have explanations that need to be considered, but we can't map each section.

Reply:
I think it would probably be good to remove this since it's the only one. That would help present the sections uniformly and prevent confusion about why only one category on this page is related to the category in the manual.